BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/view/BoardsView.java

```diff
@@ -0,0 +1,52 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.view;
+
+import pl.jakubowskiprzemyslaw.tajgertim.models.QueueObject;
+import pl.jakubowskiprzemyslaw.tajgertim.models.board.Ship;
+import pl.jakubowskiprzemyslaw.tajgertim.models.coordinates.Coordinate;
+import pl.jakubowskiprzemyslaw.tajgertim.models.player.Player;
+import pl.jakubowskiprzemyslaw.tajgertim.models.shoot.ShootResult;
+
+import java.util.List;
+import java.util.Map;
+
+public class BoardsView implements QueueObject {
```

Comment:
Tip: maybe separate "channel" for board views (coordinate -> field state) and for ships

Reply:
We will think about it... later